modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -61,4 +61,8 @@ For more information, see the link:{LinkAdminGuide}#proc-viewing-installed-plugi
 
 == Supported plugins in {product} {product-version}
 
-To get a comprehensive list of supported dynamic plugins, see the link:{LinkAdminGuide}#rhdh-supported-plugins[Dynamic plugins included in {product}] section in the _Administration guide_.
+To get a comprehensive list of supported dynamic plugins, see the link:{LinkPluginsGuide}#rhdh-supported-plugins[Dynamic plugins included in {product}] section in the _{NameOfPluginsGuide} guide_.
+
+== Compatible plugins in {product} {product-version}
+
+To get a comprehensive list of compatible dynamic plugins, see the link:{LinkPluginsGuide}#rhdh-compatible-plugins[Compatible plugins] section in the _{NameOfPluginsGuide} guide_.
```

Comment:
Suggestion: Replace lines 64 to 68 with: 

The documentation now lists multiple groups of supported plugins. See these sections in the documentation for further details:

- Red Hat Generally Available plugins
- Red Hat Technology Preview plugins
- Community plugins
- Other compatible plugins
These bulleted items could be hyperlinks to the documentation sections. (@Gerry-Forde might fine tune this when the restructured plugin docs are in place.)

Reply:
restructured plugin docs are already in place. I'll updatemy PR with the links.

Reply:
> Nick, Re. restructured plugin docs are already in place. 
@Gerry-Forde has a draft for the restructure, which could slightly tweak the changes in this PR. I will try to point out some in this review as well. We will share the draft with all of you before we merge it though.

Reply:
Draft restructure doc: https://docs.google.com/document/d/1YdQJyXc4cJrsDNvk3yjDQMvfI0gLzv6J9AGRXbQgEP4

Reply:
Full restructure doc is available [here](https://docs.google.com/document/d/1YdQJyXc4cJrsDNvk3yjDQMvfI0gLzv6J9AGRXbQgEP4/edit?usp=sharing).